### ISSUE TITLE
division of labeled part & unlabeled part of the dataset?

### ISSUE THREAD

**Issue by iTomxy** [PKU-ICST-MIPL/SCHGAN_TCYB2018#4]
Hello,
I have a question that **how is a dataset divided into labeled part & unlabeled part**?

Take NUS-WIDE as an example. As said in section *Experiments/Dataset/NUSWIDE* in the paper, you *randomly sampled 5000 images as training set*.
Does it mean that **5000 images in the retrieval set *(i.e. training set)* are used as labeled data and the rest are used as unlabeled data?**

Thanks

**Comment by PKU-ICST-MIPL** [comment]
> Hello,
> I have a question that **how is a dataset divided into labeled part & unlabeled part**?
> 
> Take NUS-WIDE as an example. As said in section _Experiments/Dataset/NUSWIDE_ in the paper, you _randomly sampled 5000 images as training set_.
> Does it mean that **5000 images in the retrieval set _(i.e. training set)_ are used as labeled data and the rest are used as unlabeled data?**
> 
> Thanks

For supervised methods, the 5000 images are used to the model with the label, the rest data have no label for the models, which are used to test the performance of the method.
